<?xml version="1.0"?>
<launch>
	<node type="guardian_node_nxt" name="guardian_node" pkg="guardian_node" output="screen">
		<param name="motor_dev" value="/dev/ttyUSB0" type="string"/>
		<param name="modbus_io_topic" value="/modbus_io/input_output" type="string"/>
		<param name="max_linear_speed" value="1.5" type="double"/> <!-- m/s -->
		<param name="max_angular_speed" value="60.0" type="double"/> <!-- degrees/s -->
		<param name="publish_tf" value="True"/>

		<!-- if use_imu=true, imu (/imu/data) will be fused in the robot odometry -->
                <param name="use_imu" value="true"/>
		<rosparam command="load" file="$(find guardian_node)/launch/guardian_conf.yaml" />
	</node>

</launch>
